src/components/GameCard.tsx

```diff
@@ -30,39 +32,53 @@ export const GameCard: React.FC<GameCardProps> = ({
   }, [game.sequelFamily]);
   const upcoming = useMemo(() => {
     if (game.releaseDate) {
-      return game.releaseDate === "TBA"
-        ? true
-        : new Date(game.releaseDate) > new Date();
-    } else {
-      return false;
+      return (
+        game.releaseDate === "TBA" || new Date(game.releaseDate) > new Date()
+      );
+    }
+
+    if (game.year) {
+      return new Date(game.year) > new Date();
     }
```

Comment:
They look great! I love the steam widget resizing and skeleton loadings, but I didn't like much the "Released" tag, I think it's not necessary as it's too repetitive, I think it's good for "Unreleased" ones to catch your attention but no more than that, you think we can correct that?

Reply:
Yeah, you're right! Removed